dashboard/app/controllers/admin_users_controller.rb

```diff
@@ -81,9 +81,9 @@ def manual_pass
     unless user
       flash[:alert] = "User (ID: #{params[:user_id]}) not found"
     end
-    script = Script.find_by_id(params[:script_id])
+    script = Script.get_from_cache(params[:script_id_or_name])
```

Comment:
Yes, this will be used on production for the "common" scripts. And now I remember why I didn't use the cache when I implemented this initially (annoying tests)... Fixing.